src/pages/Admin/Users/GroupCard/index.jsx

```diff
@@ -1,34 +1,35 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import { translate } from 'react-i18next';
 import classnames from 'classnames';
 
 import styles from './index.scss';
 
+@translate()
 export default class GroupCard extends PureComponent {
   static propTypes = {
     groups: PropTypes.array,
-    selectItem: PropTypes.number,
+    selectValue: PropTypes.string,
     selectCard: PropTypes.func
   };
 
   render() {
-    const { groups, selectCard, selectItem } = this.props;
+    const { groups, selectCard, selectValue, t } = this.props;
     return (
       <ul className={styles.groupCard}>
-        {groups &&
-          groups.map((data, index) => (
-            <li
-              key={data.id}
-              onClick={() => {
-                selectCard(index, name);
-              }}
-              className={classnames({ [styles.current]: selectItem === index })}
-            >
-              <div className={styles.name}>{data.name}</div>
-              <div className={styles.id}>id:{data.id}</div>
-              <div className={styles.description}>{data.description}</div>
-            </li>
-          ))}
+        {groups.map((data, index) => (
+          <li
+            key={data.id || index}
+            onClick={() => {
+              selectCard(data);
+            }}
+            className={classnames({ [styles.current]: data.value === selectValue })}
+          >
+            <div className={styles.name}>{t(data.name)}</div>
+            {data.id && <div className={styles.id}>id:{data.id}</div>}
```

Comment:
what if `data.id` is 0 ?

Reply:
Add jude becase roles data from  front end,  groups data from api，id is string.

Reply:
ok